FacesofLondon/app/src/main/java/com/novoda/wallpaper/droidcon/TimeOfDayCalculator.java

```diff
@@ -0,0 +1,34 @@
+package com.novoda.wallpaper.droidcon;
+
+import java.util.Calendar;
+import java.util.Locale;
+
+import static com.novoda.wallpaper.droidcon.TimeOfDay.*;
+
+class TimeOfDayCalculator {
+
+    TimeOfDay currentTimeOfDay() {
+        Calendar calendar = Calendar.getInstance();
+        int hourOfDay = calendar.get(Calendar.HOUR_OF_DAY);
+        return forHourOfDay(hourOfDay);
+    }
+
+    TimeOfDay forHourOfDay(int hourOfDay) {
```

Comment:
I think what @lgvalle meant was to move this method directly into `TimeOfDay`, so that you can call it in `currentTimeOfDay()` by doing something like `TimeOfDay.fromHourOfDay(hour)`